packages/clarity-js/src/layout/dom.ts

```diff
@@ -334,6 +351,26 @@ export function updates(): NodeValue[] {
     return output;
 }
 
+export function getFragments(): {} {
+    let output = {};
+    if (fragments) {
+        fragments.forEach(hash => {
```

Comment:
Minor: Just use the null coalescing operators like ?., ?[ and ?( for code like this and saves the if() checks